### One Tailed (Left Tailed) - One-Sample Z-Test

**1 - What is it?**

A. `Left One-Tailed One-Sample Z-Test` is used to test whether a sample mean is `significantly lower` than population mean, when the `population standard deviation` is known and the `sample size is large (typically>=30)`.

**In other words, it checks:**

Is my sample average *truly smaller* than the overall population average, or is that difference just due to random variation?

**Example**

Is my small group's of average scores *really lower (significantly)* from the entire country's average score, or it just random chance.

**2-Real-Life Example: Left Tailed One-Sample Z-Test**

`Scenario:` A teacher believes here new teaching method might have produced `lower scores` this year due to a change in syllabus.

So we wanted to test if its really decreased the scores, not just difference - So it's one-tailed test (specifically, right-tailed)

**3-Hypotheses**

- H0 (Null Hypothesis) : No decrease in population mean
- H1 (Alternate Hypothesis) : Sample mean < Population mean (scores decreased)

This is `left-tailed` test because we're checking `less than`

**4 - Formula & Parameters**

Z = (Sample Mean - Population Mean) / (Population Std)/Sqrt(Sample Size)

Z = (x bar - μ) / (σ/sqrt(n))

**Parameters:**

- Population mean (μ) : 70
- Population std (σ) : 10
- Sample mean (x bar) : 67
- Sample size (n) : 40
- Significance level (a) : 0.05
- Confidence Interval (CI) : 0.95

Here, `populatin std is known and sample size >=30`, So we can do z-test, specifically *left one-tailed one-sample z-test*.

**5 - Python Implementation (Step-by-Step)**

In [6]:
import numpy as np
from scipy import stats

# Step 1: Define Parameters
population_mean = 70    # Known population mean
population_std = 10     # Known population standard_deviation
sample_mean = 67        # Sample mean
sample_size = 40        # Sample size
alpha = 0.05            # Significance level

# Step 2: Calculate Standard Error (SE)
standard_error = population_std/np.sqrt(sample_size)
print(f"Standard Error (SE): {standard_error:.4f}")

# Step 3: Compute Z Statistics
z_stat = (sample_mean - population_mean) / standard_error
print(f"Z Statistic: {z_stat:.4f}")

# Step 4: Compute P-Value for left One-Tailed Test
# For a left-tailed test, we take area to the left of z_stat
p_value = stats.norm.cdf(z_stat)
print(f"P-Value: {p_value:.4f}")

# Step 5: Compute Critical Value (Boundary of Rejection Region)
# For left-tailed test, critical value = z_(alpha)
critical_value = stats.norm.ppf(alpha)
print(f"Critical Value: {critical_value:.4f}")

# Step 6: Decision Making
print("\n--- Decision Based on P-Value ---")
if p_value < alpha:
    print("Reject H0: The sample mean is significantly lower than the population mean.")
else:
    print("Fail to Reject H0: No significant evidence that the sample mean is lower.")

print("\n--- Decision Based on Critical Value ---")
if z_stat < critical_value:
    print("Reject H0: The sample mean is significantly lower than the population mean.")
else:
    print("Fail to Reject H0: No significant evidence that the sample mean is lower.")

Standard Error (SE): 1.5811
Z Statistic: -1.8974
P-Value: 0.0289
Critical Value: -1.6449

--- Decision Based on P-Value ---
Reject H0: The sample mean is significantly lower than the population mean.

--- Decision Based on Critical Value ---
Reject H0: The sample mean is significantly lower than the population mean.


**6 - Business Interpretation**

- **Z = 1.90** -> The sample mean is 1.9 standard error below the population mean.
- **p = 0.029** -> There's only about a 2.9% probability of observing such a low average if the true mean were still 70.

**Conclusion:**
This suggests that decrease in score is **statistically significant**, meaning it's unlikely due to random variation.
The new syllabus **likely had a negative impact** on students performance.